### PR TITLE
fix: updated example with version number

### PR DESCRIPTION
**Pull request by OndrejTuma**
### Description

- add (required) version number to the example of `report-affected-packages`
- add lint for changeset files

### How to test

- checkout branch
- generate empty changeset, add it to staged area and run lint-staged: 
`yarn changeset --empty && git add .changeset && yarn lint-staged`

### Development checks

- Add changeset according to [guidelines](https://github.com/toptal/davinci-github-actions/blob/master/_docs/contribution/changeset-guidelines.md) (if needed)

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-anvil ping reviewers` - Ping FX team for review

</details>

### CODE REVIEW

File: report-affected-packages/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## Report affected packages
 
-Generate a diagram of affected monorepo packages as a job summary. Works with worksflows triggered by pull\_request event.
+Generate a diagram of affected monorepo packages as a job summary. Works with workflows triggered by pull\_request event.
```
Comment (separatio) [inline report-affected-packages/README.md:3]: Nice catch. :smile:

**Review by separatio — APPROVED** [review]
Didn't test. Code looks good. Nice catch with the worksflows. :smile:

**Review by denieler — APPROVED** [review]
Haven't tested, code looks good and it should work 😃 👍

**Comment by OndrejTuma** [comment]
@toptal-anvil ping reviewers

**Review by dmaklygin — APPROVED** [review]

**Pull request merged**